packages/app-store/ee/routing-forms/pages/forms/[...appPages].tsx

```diff
@@ -27,12 +32,49 @@ export default function RoutingForms({
   appUrl,
 }: inferSSRProps<typeof getServerSideProps> & { appUrl: string }) {
   const { t } = useLocale();
-  const { data: forms } = trpc.viewer.appRoutingForms.forms.useQuery(undefined, {
+
+  const { data: forms, isLoading } = trpc.viewer.appRoutingForms.forms.useQuery(undefined, {
     initialData: forms_,
   });
 
+  const { data } = trpc.viewer.teams.list.useQuery(); // TODO: use hasTeam hook
+  const teams = useMemo(() => data?.filter((m) => m.accepted) || [], [data]); // TODO: use hasTeam hook
+
```

Comment:
@CarinaWolli turn this into a hook and in all other places where we check for teams not empty

Reply:
working on it right now